views/pages/addComment.ejs

```diff
@@ -65,8 +68,15 @@ getBreadcrumb = function () {
         
         var data = serialiseForm(frm);
 
+        <% 
+            var requestType = "POST";
+            if (typeof comment != "undefined") { 
```

Comment:
Handling creates and updates with the same template creates a lot of convoluted logic here. If you do that you need to generate javascript and add some if-ology (note that google analytics event is also set to add right now). Think it would be best to create a separate template even though it will be similar - it would be easier to maintain in the future (setting different page titles for add and update, doing different styles etc.)